ReflectometryServer/pv_wrapper.py

```diff
@@ -518,6 +518,9 @@ def _on_update_velocity(self, value, _alarm_severity, _alarm_status):
             alarm_status (server_common.channel_access.AlarmCondition): the alarm status
         """
         self._velocity_cache = value
+        if self._velocity_restored or (self._velocity_restored is None and not self.is_moving):
```

Comment:
```suggestion
        velocity_unrestored_and_not_moving = (self._velocity_restored is None and not self.is_moving)
        if self._velocity_restored or velocity_unrestored_and_not_moving:
```

Maybe this is clearer, if not feel free to ignore